packages/fcl/CHANGELOG.md

```diff
@@ -3,6 +3,58 @@
 - YYYY-MM-DD **BREAKING?** -- description
 
 - 2021-10-29 -- Removes default `config.fcl.appDomainTag` and updates docs.
+- 2021-11=05 -- Allow configurable Discovery in FCL via `fcl.discovery` with current support for authn services.
+
+An app developer will now be able to list services for authentication ("authn") in their application and authenticate without having to point directly at a single wallet or `"discovery.wallet"` which is a pre-constructed UI for Flow services.
+
+To use this, first set in the config the API endpoint for fetching Flow services.
+
+```javascript
+import { config } from "@onflow/fcl"
+
+config({
+  "discovery.authn.api": "https://fcl-discovery.onflow.org/api/testnet/authn"
```

Comment:
I'm partial to `discovery.authn.endpoint` for this key

Reply:
updated